src/main/webapp/i18n/en/course.json

```diff
@@ -23,6 +23,22 @@
             "color": "Color",
             "courseIcon": "Course Icon",
             "shortName": "Short Name",
+            "maxComplaints": {
+                "title": "Maximum Amount of Complaints",
+                "description": "Set the maximum amount of complaints allowed"
+            },
+            "maxComplaintTimeDays": {
+                "title": "Deadline for complaints in days after result date",
+                "description": "Maximum Complaint Deadline for student complaints. When the complaint is submitted after the deadline, it will be rejected."
```

Comment:
this is not 100% correct. After this period, students won't be able to complain any more.
Also please mention that the period refers to days after the result is available to the student (i.e. after the assessment due date or after the result date, depending on which of them is later)